planner_cspace/src/planner_3d.cpp

```diff
@@ -1322,6 +1325,43 @@ class Planner3dNode
       {
         return;
       }
+      else if (is_path_switchback_)
+      {
+        static ros::Time last_time = start_.header.stamp;
+        const ros::Time time = start_.header.stamp;
+        const float dt = time.toSec() - last_time.toSec();
+        if (dt > 0.0)
```

Comment:
value of `dt` is not used.
```suggestion
        if (time > last_time)
```

Reply:
Time jump back should be always checked, not only on `is_path_switchback_`.
Moving it to the outer scope would be better.